pages/blog/[seoTitleAndId].js

```diff
@@ -34,7 +34,10 @@ export async function getStaticPaths() {
 }
 
 export async function getStaticProps({ params }) {
-    const postId = params.id;
+    const seoTitleAndId = params.seoTitleAndId;
+    const parts = seoTitleAndId.split('-');
+    const postId = parts[parts.length - 1];
```

Comment:
# Reviewer Note

@eang0521 @tang0226 FYI -- this was the key part -- sending in the SEO title and blog post ID as a compound value in one slug, and then extracting the parts later.